pkg/services/ngalert/api/generated_base_api_prometheus.go

```diff
@@ -1,17 +1,14 @@
-/*Package api contains base API implementation of unified alerting
+/* Package api contains base API implementation of unified alerting
```

Comment:
CI complains with `https://revive.run/r#package-comments  package comment should not have leading space` because of this change